### ISSUE TITLE
Add disassembly "decompiler"

### ISSUE THREAD

**Issue by dougallj** [decompiler-explorer/decompiler-explorer#50]
Currently if you have a dogbolt link where two tools disagree, there's no way to know which one is correct. (e.g. BinaryNinja vs Hex-Rays [here](https://dogbolt.org/?id=ab5811dc-04a3-45e1-91df-3233c5583b33))

Adding a "disassembly" pane would give a source of truth, to allow users to understand what the input to the decompilers actually was, and which one is correct.

**Comment by CouleeApps** [comment]
This is a good idea although it is more nuanced than "just slap objdump on it," considering there are many different disassemblers for many platforms and they often disagree. We could add runners for many different disassemblers for comparison between them though.

**Comment by dougallj** [comment]
Yeah - for now I'd lean towards "just slap objdump on it" (and call it objdump) as an easy and mostly-fine tradeoff? (Free for devs without binja licenses, and faster than ghidra)

On the other extreme, I did also prototype dumping out disassembly with decompilation line-number associations from Hex-Rays, displayed with godbolt style colouring of the disassembly and pseudocode, which might be possible with other tools too. (I think setting `WITH_DISASM = True` on line 5 of [dump.py](https://gist.github.com/dougallj/87695462d6200714d6029366fa80e98f#file-dump-py) worked)
